homeassistant/components/motion_blinds/config_flow.py

```diff
@@ -72,6 +74,15 @@ def async_get_options_flow(config_entry) -> OptionsFlowHandler:
         """Get the options flow."""
         return OptionsFlowHandler(config_entry)
 
+    async def async_step_dhcp(self, discovery_info: dhcp.DhcpServiceInfo) -> FlowResult:
+        """Handle discovery via dhcp."""
+        mac_address = format_mac(discovery_info.macaddress).replace(":", "")
+        await self.async_set_unique_id(mac_address)
+        self._abort_if_unique_id_configured(updates={CONF_HOST: discovery_info.ip})
+
+        self._host = discovery_info.ip
```

Comment:
We should probably set title_placeholders here as well

Reply:
If you don't have a name you can do something like

AABBCC (192.168.1.11)

Where AABBCC is the last 6 of the Mac

Reply:
Done